src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameCheckEnhancedInstanceof.java

```diff
@@ -0,0 +1,29 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+public class InputAbbreviationAsWordInNameCheckEnhancedInstanceof {
```

Comment:
there should be config of check as comment, please fin other Inputs for example.

Reply:
https://github.com/checkstyle/checkstyle/pull/8437